### PR TITLE
CASMINST-3004

### PR DESCRIPTION
**Pull request by lukebates123**
Remove unnecessary instructions related to BGP that may cause confusion.

### CODE REVIEW

**Comment by johren-hpe** [comment]
This is removing the step to clear the BGP sessions.   We still need those steps included.

File: install/deploy_management_nodes.md [hunk 1]
```diff
@@ -571,59 +571,6 @@ The LiveCD needs to authenticate with the cluster to facilitate the rest of the
...
-    - Aruba: `show bgp ipv4 unicast summary`
-    - Mellanox: `show ip bgp summary`
-
-    You should see a neighbor for each of the workers NCN IP addresses found in an earlier step. If it is an Aruba switch, you will also see a neighbor for the other switch of the pair that are peering.
-
-   At this point the peering sessions with the worker IP addresses should be in `IDLE`, `CONNECT`, or `ACTIVE` state (not `ESTABLISHED`). This is due to the MetalLB speaker pods not being deployed yet.
-
-   You should see that the `MsgRcvd` and `MsgSent` columns for the worker IP addresses are 0.
-
-1. If the neighbor IP addresses do not match the worker NCN IP addresses, use the helper script for Mellanox and CANU (Cray Automated Network Utility) for Aruba.
```
[the remark this answers is not in the document]
Comment (johren-hpe) [reply install/deploy_management_nodes.md:614]: Response from Luke in slack:
when the bgp script runs it wipes out the previous bgp config and then overwrites, so if we’re having them do that everytime I think the clear BGP neighbors shoudn’t be necessary.

[the remark this answers is not in the document]
Comment (johren-hpe) [reply install/deploy_management_nodes.md:614]: If that is the case that we always wipe out the previous config, I think we should be okay regarding BGP session reset.   
However, I think we should still have the steps above to tell them how to find the IP addresses for the switches, right?

[the remark this answers is not in the document]
Comment (johren-hpe) [reply install/deploy_management_nodes.md:614]: We also need SYSTEM_NAME for the paths arguments in the helper script and CANU commands below.

[the remark this answers is not in the document]
Comment (lukebates123) [reply install/deploy_management_nodes.md:614]: Added back the steps to get the switch IPs.

File: install/deploy_management_nodes.md [hunk 2]
```diff
@@ -571,57 +571,22 @@ The LiveCD needs to authenticate with the cluster to facilitate the rest of the
...
+      pit# export SYSTEM_NAME=eniac
+      ```
 
-1. If the neighbor IP addresses do not match the worker NCN IP addresses, use the helper script for Mellanox and CANU (Cray Automated Network Utility) for Aruba.
+   1. Determine the IP address of the worker NCNs.
 
-   1. This command will list the available helper scripts.
       ```bash
-      pit# ls -1 /usr/bin/*mellanox_set_bgp_peer*py
+      pit# grep -B1 "name: ncn-w" /var/www/ephemeral/prep/${SYSTEM_NAME}/networks/NMN.yaml
```
Comment (johren-hpe) [inline install/deploy_management_nodes.md:583]: Do we need the worker NCN IPs still?

Comment (johren-hpe) [reply install/deploy_management_nodes.md:583]: I see now after expanding below that we need them for the step of checking the status.

File: install/deploy_management_nodes.md [hunk 3]
```diff
@@ -571,57 +571,22 @@ The LiveCD needs to authenticate with the cluster to facilitate the rest of the
...
-
-1. Determine the IP addresses for the switches that are peering.
-
-   ```bash
-   pit# grep peer-address /var/www/ephemeral/prep/${SYSTEM_NAME}/metallb.yaml
-   ```
-
-1. Do the following steps for each of the switch IP addresses that you found in the previous step:
-
-    1. Log in to the switch as the `admin` user:
```
Comment (johren-hpe) [inline install/deploy_management_nodes.md:594]: I think we'll need this login step below where we are telling them to check the status on the switch (after running the helper script or canu).

**Review by johren-hpe — CHANGES_REQUESTED** [review]

**Review by johren-hpe — APPROVED** [review]

**Review by denniswalker — APPROVED** [review]

**Pull request merged**